tools/defaults.bzl

```diff
@@ -400,3 +405,57 @@ def ng_web_test_suite(deps = [], static_css = [], exclude_init_script = False, *
         bootstrap = bootstrap,
         **kwargs
     )
+
+def spec_bundle(name, deps, **kwargs):
```

Comment:
Is there a compelling reason not to have this extraction occur in `spec_bundle` from `dev-infra`? Or even as a boolean attr?  I can't think of a situation where we would want to `spec_bundle` and build both.

Reply:
The issue is that the COMP repo is the only repo building devmode as ESM currently, so this does not apply at all to other repositories.

A flag would work, but I'd rather prefer to keep it local until we have decided on a plan for combining devmode & prodmode. does that sound reasonable?

Reply:
Ahh yes, that makes sense.  Yeah, we can just leave it as is and once and address it once we have the larger plan in place.